### DISCUSSION TITLE
EVGA store needs notification for ability to AutoNotify, instead of ORDER

### DISCUSSION THREAD

**Discussion by TunaErtemalp** [jef/streetmerchant#792] [Ideas and feature requests]
See https://twitter.com/EVGA_JacobF/status/1327298202177990656. Yes, a while ago they had converted to a queue model with AutoNotify leading to individual future order windows, but an hour ago their game got further twisted! They introduced the 3080/3090 Hybrid & HydroCopper cards, put the AutoNotify buttons up, sent the tweet, and then when they sold out of that inventory, they removed the buttons. If you didn't see the tweet and act within 30-40 minutes, you lost out. Which means, StreetMerchant would have been great to scrape for the AutoNotify button being available and alert me before I saw the tweet. And, to make things more complicated, they are NOT doing this for their non-water cards, leaving the AutoNotify button up for them, so any scraping for AutoNotify on EVGA cards would need to be adjustable at URL level so that the other cards don't generate notification after notification after notification.

**Comment by jxjftw** [comment]
Here's the problem with that... SM monitors individual page links, not entire search pages, you'd have to have had the individual page links to notify on the new button anyways which we didn't have until the button went live.
